### PR TITLE
WIP Octane upgrade

### PR DESCRIPTION
**Pull request by vaidehijoshi**

### CODE REVIEW

**Comment by jenweber** [comment]
We determined in chat that the issue was that we need to enable jquery

File: package.json [hunk 1]
```diff
@@ -18,33 +18,37 @@
...
+    "ember-data": "^3.10.0",
     "ember-export-application-global": "^2.0.0",
-    "ember-load-initializers": "^1.1.0",
+    "ember-load-initializers": "^2.0.0",
     "ember-maybe-import-regenerator": "^0.1.6",
-    "ember-qunit": "^3.4.1",
-    "ember-resolver": "^5.0.1",
+    "ember-qunit": "^4.1.2",
+    "ember-resolver": "^5.1.3",
     "ember-simple-leaflet-maps": "^0.0.2",
```
Comment (rwjblue) [inline package.json:47]: I think if we update this, we shouldn't need jQuery (unless there is another plugin that is requiring it)

Crosslinking from https://github.com/ember-learn/ember-simple-leaflet-maps/issues/2

**Comment by vaidehijoshi** [comment]
Update on this: @wycats and I are pairing on this together and taking another approach. We're building the super-rentals app from scratch using the Octane blueprint and making note of our steps along the way that we can use for the upgrade guide once we have a completely idiomatic version of the super-rentals app.

**Comment by jenweber** [comment]
@vaidehijoshi @wycats any updates?

**Comment by vaidehijoshi** [comment]
We still have a bit more work to do on the app, but not _too_ much more. We should hopefully be able to pair on it for another hour or two next week and get the Octane-idiomatic version of super-rentals wrapped up! (cc @wycats)

**Comment by vaidehijoshi** [comment]
Since we've changed our strategy on how to go about doing the work of upgrading this app, I'm going to close this PR in favor of #127. Please see that PR for my work with @wycats on making super-rentals Octane-idiomatic :)

**Pull request closed without merging**